MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -4609,13 +4354,434 @@ private void fillRankInfo() {
+        spnAtBBattleChance[AtBLanceRole.SCOUTING.ordinal()].setValue(options.getAtBBattleChance(AtBLanceRole.SCOUTING));
+        spnAtBBattleChance[AtBLanceRole.TRAINING.ordinal()].setValue(options.getAtBBattleChance(AtBLanceRole.TRAINING));
+        btnIntensityUpdate.doClick();
+        chkGenerateChases.setSelected(options.generateChases());
+
+        btnDynamicRATs.setSelected(!options.useStaticRATs());
+        btnStaticRATs.setSelected(options.useStaticRATs());
+        for (String rat : options.getRATs()) {
+            List<Integer> eras = RATManager.getAllRATCollections().get(rat);
+            if (eras != null) {
+                StringBuilder displayName = new StringBuilder(rat);
+                if (eras.size() > 0) {
+                    displayName.append(" (").append(eras.get(0));
+                    if (eras.size() > 1) {
+                        displayName.append("-").append(eras.get(eras.size() - 1));
+                    }
+                    displayName.append(")");
+                }
+                if (availableRatModel.contains(displayName.toString())) {
+                    chosenRatModel.addElement(displayName.toString());
+                    availableRatModel.removeElement(displayName.toString());
+                }
+            }
+        }
+        chkIgnoreRatEra.setSelected(options.canIgnoreRatEra());
+
+        chkDoubleVehicles.setSelected(options.getDoubleVehicles());
+        spnOpforLanceTypeMechs.setValue(options.getOpforLanceTypeMechs());
+        spnOpforLanceTypeMixed.setValue(options.getOpforLanceTypeMixed());
+        spnOpforLanceTypeVehicles.setValue(options.getOpforLanceTypeVehicles());
+        chkOpforUsesVTOLs.setSelected(options.getOpforUsesVTOLs());
+        chkOpforUsesAero.setSelected(options.getAllowOpforAeros());
+        spnOpforAeroChance.setValue(options.getOpforAeroChance());
+        chkOpforUsesLocalForces.setSelected(options.getAllowOpforLocalUnits());
+        spnOpforLocalForceChance.setValue(options.getOpforLocalUnitChance());
+        chkAdjustPlayerVehicles.setSelected(options.getAdjustPlayerVehicles());
+        chkRegionalMechVariations.setSelected(options.getRegionalMechVariations());
+        chkAttachedPlayerCamouflage.setSelected(options.getAttachedPlayerCamouflage());
+        chkPlayerControlsAttachedUnits.setSelected(options.getPlayerControlsAttachedUnits());
+        chkUseDropShips.setSelected(options.getUseDropShips());
+        chkUseWeatherConditions.setSelected(options.getUseWeatherConditions());
+        chkUseLightConditions.setSelected(options.getUseLightConditions());
+        chkUsePlanetaryConditions.setSelected(options.getUsePlanetaryConditions());
+        chkUseAtBCapture.setSelected(options.getUseAtBCapture());
+        spnStartGameDelay.setValue(options.getStartGameDelay());
+        //endregion Against the Bot Tab
+
+        //region Miscellaneous Tab
+        chkHistoricalDailyLog.setSelected(options.historicalDailyLog());
```

Comment:
This probably ought to go into MekHQ preferences, but not for this PR.

Reply:
On my current list to do, just not in this PR.